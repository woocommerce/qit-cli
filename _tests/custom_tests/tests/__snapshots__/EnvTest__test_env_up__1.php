<?php return 'Warning: Key "skip_activating_plugins" not found in environment info.
Setting up Docker...
Setting up WordPress...
Activating plugins...

 [OK] Temporary test environment created. (662ac7d2235f9)                       

 * URL: http://localhost:32776
 * Admin URL: http://localhost:32776/wp-admin
 * Admin Credentials: admin/password
 * PHP Version: 7.4
 * WordPress Version: 6.5.2
 * Redis Object Cache? No
 * Path: /home/lucas/automattic/qit-cli/_tests/custom_tests/tmp_qit_config/temporary-envs/e2e-662ac7d2235f9/

To see additional info, run with the "--verbose" flag.

http://localhost:32776
';
