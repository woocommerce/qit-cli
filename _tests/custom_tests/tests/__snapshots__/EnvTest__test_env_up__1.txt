Setting up Docker...
First-time setup is pulling Docker images and caching downloads. Subsequent runs will be faster.
Setting up WordPress...
Activating plugins...

 [OK] Temporary test environment created. (ENV_ID)                       

 * URL: http://localhost:PORT
 * Admin URL: http://localhost:PORT/wp-admin
 * Admin Credentials: admin/password
 * PHP Version: 7.4
 * WordPress Version: NORMALIZED
 * Redis Object Cache? No
 * Path: RUN_ID/temporary-envs/e2e-ENV_ID/

To see additional info, run with the "--verbose" flag.

http://localhost:PORT
